spec/features/staff/users_can_view_an_activity_as_xml_spec.rb

```diff
@@ -118,15 +118,15 @@
       end
 
       context "when the activity is a fund activity" do
-        let(:activity) { create(:fund_activity, organisation: organisation, identifier: "IND-ENT-IFIER") }
+        let(:activity) { create(:fund_activity, organisation: organisation, identifier: "IND-ENT-IFIER", transparency_identifier: "GB-GOV-13-IND-ENT-IFIER") }
```

Comment:
Seeing the first change made to the factory makes me question whether we should do this type of work automatically inside FactoryBot, if possible?

```
transparency_identifier: { "#{parent.iati-identifier}-#{identifier}" }
```

Can we reference the parent like this or would we have to do it in an after create blob?

Reply:
Done with a trait & after create hook!